charts/hivemq-operator/templates/global/hivemq-psp.yaml

```diff
@@ -31,25 +29,25 @@ spec:
   hostPID: false
   runAsUser:
     # Permits the container to run with root privileges as well.
-    rule: 'RunAsAny'
+    rule: 'MustRunAsNonRoot'
```

Comment:
The comment above the line would need adjustment, as we are now explicitly forbid to run as root